config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
```

Comment:
Why is it `draw`?

Reply:
Because thats the method name that the guy who wrote it decide to be 😅